lute/std/libs/time.luau

```diff
@@ -13,6 +17,8 @@ type durationdata = {
 	_nanoseconds: number,
 }
 
+local time = {}
+
 local duration = {}
 duration.__index = duration
 
```

Comment:
As discussed irl, we should move this into a proper submodule in a follow-up PR and re-export here.